tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -80,7 +83,33 @@ sub run {
 
     # Close Firefox
     quit_firefox;
+
+    wait_still_screen 60 if is_aarch64;
     assert_screen("generic-desktop", 20);
+    sleep 30;
```

Comment:
Hi Ben, 

I would suggest don't use a straight sleep 30 here :)

Reply:
yes, I will change to another way, thanks.

Reply:
Done!